CHANGELOG.md

```diff
@@ -30,6 +32,9 @@ raw binaries. All binaries have OS in their names as well now, following
 regular naming used throughout NSPCC, so instead of neofs-cli-amd64 you get
 neofs-cli-linux-amd64 now.
 
+CLI command `acl extended create` changed and extended input format for filters.
+For example, `attr>=100` or `attr=` are now processed differently. See `-h` for details.
```

Comment:
"See `-h`"? see help? use -h flag?

Reply:
see help text

Reply:
?

Reply:
`-h` prints help text that u can see

Reply:
i mean i cant remember i see somewhere "see -h". it is usually "see help" or "use -h flag for details"

Reply:
i bet anybody will understand what is this about